sdk/dns/arm-dns/package.json

```diff
@@ -24,23 +24,23 @@
     "isomorphic"
   ],
   "license": "MIT",
-  "main": "./dist/index.js",
-  "module": "./dist-esm/src/index.js",
-  "types": "./types/arm-dns.d.ts",
+  "main": "./dist/commonjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/commonjs/index.d.ts",
   "devDependencies": {
-    "typescript": "~5.7.2",
-    "dotenv": "^16.0.0",
+    "@azure-tools/test-credential": "^2.0.0",
+    "@azure-tools/test-recorder": "^4.1.0",
+    "@azure-tools/test-utils-vitest": "^1.0.0",
     "@azure/dev-tool": "^1.0.0",
     "@azure/identity": "^4.2.1",
-    "@azure-tools/test-recorder": "^3.0.0",
-    "@azure-tools/test-credential": "^1.1.0",
-    "mocha": "^11.0.2",
-    "@types/mocha": "^10.0.0",
-    "tsx": "^4.7.1",
-    "@types/chai": "^4.2.8",
-    "chai": "^4.2.0",
     "@types/node": "^18.0.0",
-    "ts-node": "^10.0.0"
+    "@vitest/browser": "^2.1.8",
+    "@vitest/coverage-istanbul": "^2.1.8",
+    "dotenv": "^16.0.0",
+    "playwright": "^1.49.1",
+    "tsx": "^4.7.1",
```

Comment:
We should probably remove tsx as well.  Just a note for the future.

Reply:
@kazrael2119 could you help create issue in SDK repo for this? Maybe could assign it to @jeremymeng or we can take a look when we are back from holiday ?

Reply:
create an issue to track https://github.com/Azure/azure-sdk-for-js/issues/32731